spec/changelog.yml

```
@@ -7,6 +7,9 @@
   - description: Prepare for next version
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/509
+  - description: Add new subcategories
```

Given that we are adding a single subcategory, could we be more specific here?
```suggestion
  - description: Add new subcategory "advanced_analytics_ueba"
```

---

Thanks for the suggestion!